### PR TITLE
fix for Falcon 1.0.0 and 1.1.0

### PR DESCRIPTION
**Pull request by kojimaeiji**
this fix should also work for 0.3.0

### CODE REVIEW

File: falcon_multipart/middleware.py [hunk 1]
```diff
@@ -17,7 +17,10 @@ def process_request(self, req, resp, **kwargs):
         # This must be done to avoid a bug in cgi.FieldStorage.
         req.env.setdefault('QUERY_STRING', '')
 
-        form = self.parse(stream=req.stream, environ=req.env)
+        if hasattr(req.stream, 'stream'):
```
Comment (yohanboniface) [inline falcon_multipart/middleware.py]: I'd do it a bit differently, something like:

    stream = req.stream.stream if hasattr(req.stream, 'stream') else req.stream
    form = self.parse(stream=stream, environ=req.env)

Plus I'd add a small comment to explain why we need this check.

**Review by yohanboniface — CHANGES_REQUESTED** [review]
Thanks!
I made a small review comment :)

File: falcon_multipart/middleware.py (hunk 1 above)
Comment (kojimaeiji) [reply falcon_multipart/middleware.py]: I changed the code as you show and added a small comment for explanation above the code.

**Review by yohanboniface — APPROVED** [review]

**Comment by yohanboniface** [comment]
Thanks! :)

**Pull request merged**